api/api.yaml

```diff
@@ -166,9 +166,12 @@ definitions:
     description: >
       Can be a field (Gender), or a field and selected value (Gender = female).
     properties:
-      display_text:
+      facet_name:
         type: string
-        description: The text displayed in the dropdown.
+        description: The name of the facet.
+      facet_description:
+        type: string
+        description: The description of the facet.
```

Comment:
Lines 181-182:

           If this search result represents a facet, facet_value is the empty string.
           If this search result represents a facet and selected value, facet_value is the selected value.